gnome-extra/gnome-shell-frippery/gnome-shell-frippery-45.0.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Unofficial extension pack providing GNOME 2-like features"
+HOMEPAGE="http://frippery.org/extensions/index.html"
+SRC_URI="http://frippery.org/extensions/${P}.tgz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	app-eselect/eselect-gnome-shell-extensions
+	>=dev-libs/gjs-1.29
+	dev-libs/gobject-introspection:=
+	gui-libs/gtk:4[introspection]
+	gnome-base/gnome-menus:3[introspection]
+	=gnome-base/gnome-shell-44*
+	media-libs/clutter:1.0[introspection]
+	x11-libs/pango[introspection]
+"
+DEPEND=""
+
+S="${WORKDIR}/.local/share/gnome-shell"
```

Comment:
The empty `DEPEND` variable could be removed + `S` should go below `SRC_URI`

Reply:
Please don't remove empty explicitly set DEPEND="" in gnome@ packages. It means it's explicitly empty, not that it's been forgotten about.

Reply:
> Please don't remove empty explicitly set DEPEND="" in gnome@ packages. It means it's explicitly empty, not that it's been forgotten about.

Oh, ok i didn't know. Thanks for explaining.

Reply:
Corrected in https://github.com/gentoo/gentoo/pull/33339/commits/973d62f9d4131fbb6603730c882328b385fb9e69